test/test-cases/functional/saic/test_vm_to_vm_commn_acl_outbound.py

```diff
@@ -0,0 +1,183 @@
+import json
+from pathlib import Path
+from pprint import pprint
+import time
+import pytest
+import sys
+sys.path.append("../utils")
+import snappi_utils as su
+
+current_file_dir = Path(__file__).parent
+
+"""
+This covers following scenario :
+
+Test vnet to vnet communication with ACL on outbound direction:
+1. Configure DUT to deny and allow traffic
+2. Configure TGEN traffic flow as one vnet to another vnet of two OpenTrafficGenerator ports
+3. Verify Traffic denied through deny traffic IPs
+
+Topology Used :
+
+       --------          -------          -------- 
+      |        |        |       |        |        |
+      |        |        |       |        |        |
+      |  TGEN  |--------|  DUT  |--------|  TGEN  |
+      |        |        |       |        |        |
+      |        |        |       |        |        |
+       --------          -------          -------- 
+       
+"""
+
+###############################################################
+#                  Declaring Global variables
+###############################################################
+
+TOTALPACKETS = 1000
+PPS = 100
+TRAFFIC_SLEEP_TIME = (TOTALPACKETS / PPS) + 2 
+PACKET_LENGTH = 128
+ENI_IP = "1.1.0.1"
+NETWORK_IP2 = "1.128.0.2"
+NETWORK_IP1 = "1.128.0.1"
+DPU_VTEP_IP = "221.0.0.2"
+ENI_VTEP_IP = "221.0.1.11"
+NETWORK_VTEP_IP = "221.0.2.101"
+OUTER_SRC_MAC = "80:09:02:01:00:01"
+OUTER_DST_MAC = "c8:2c:2b:00:d1:30" 
+INNER_SRC_MAC = "00:1A:C5:00:00:01"
+INNER_DST_MAC = "00:1b:6e:14:00:02"
+
+
+###############################################################
+#                  Start of the testcase
+###############################################################
+
+class TestAclOutbound:
+
+    @pytest.fixture(scope="class")
+    def setup_config(self):
+        """
+        Fixture returns the content of the file with SAI configuration commands.
+        scope=class - The file is loaded once for the whole test class
+        """
+        current_file_dir = Path(__file__).parent
+        with (current_file_dir / 'config_outbound_setup_commands.json').open(mode='r') as config_file:
+            setup_commands = json.load(config_file)
+        return setup_commands
+
+    def test_setup(self, dpu, setup_config):
+        results = [*dpu.process_commands(setup_config)]
+        print("\n======= SAI setup commands RETURN values =======")
+        pprint(results)
+        assert all(results), "Setup error"
+
+    def test_vm_to_vm_commn_acl_outbound(self, dataplane):
```

Comment:
I'd recommend to make it dependent on `test_setup`. Something like here https://github.com/opencomputeproject/SAI-Challenger/blob/3f6e90c5f5283782abcc21261861d369752144f2/tests/ut/test_acl_ut.py#L84

Reply:
Thanks for your feedback. Addressed this comment.